make_european.py

```diff
@@ -0,0 +1,82 @@
+"""Take input in the format:
+mm/dd/yyyy:username@host
+Print out all of the usernames that have the host 'aol' and convert
+the date to European format so that the following format is produced:
+dd/mm/yyyy username
+"""
+import re
+
+def get_input():
+    """Prompts the user for input in the format:
+    mm/dd/yyyy:username@host
+    Receive the input as raw_input.
+    """
+    try:
+        return raw_input()
+    except EOFError:
+        return ' '
+
+def convert_input(string, desired_host):
+    """Take in a string and a desired host. If the host found in the
+    string matches the desired host, then convert the found date to the
+    European format and convert the username to all lowercase. Return
+    the new date and username.
+    """
+    assert isinstance(string, str), "Initial input for convert_input needs to be a string."
```

Comment:
We worked on these for awhile so you could get the hang of asserts, but actually type checking like this in python or ruby is considered 'unpythonic'. For instance, there might be other objects that are 'string-like' enough for your code to work on that these assertions would bar.

Are there other assertions you can add, either at the top here or in your code, that asserts more on the *formatting* of the string, for instance?